index.html

```diff
@@ -33,11 +34,8 @@ <h2 class="headline inverted uppercase program__title">Yoga</h2>
 
       <div class="meta uppercase">
         <p class="meta__item">
-          <span class="unbreakable">Wednesdays 6:00 — 7:30pm</span>
-          <span class="unbreakable">@ Unity Spiritual Center</span>
+          <span class="unbreakable"><a href="beginners-yoga-course.html">Sign up here for the Beginners Yoga Course starting April 13th!</a></span>
```

Comment:
I'd also argue that as the primary call to action on the site, this doesn't stand out enough. To me it seems like a banner treatment might be more appropriate where it appears above the "Yoga" and "Reiki" headlines, spanning both columns. There are a lot of different ways to approach this, I just think that it needs more oomph.